micro-benches/0-level/openmp/ordering/comm_free.c

```diff
@@ -4,14 +4,15 @@
 #include <stdbool.h>
 #include <stdlib.h>
 
+// Using a freed communicator is not allowed:
+// Before a thread finishes the sendrecv (marker "A"), due to the nowait, another thread may execute comm_free
+// on the communicator (marker "B").
+
```

Comment:
meabe rephrase it: strictly speaking: the Ordering sendrecv -> free is allowed (dont care if comm actually finishes, MPI mandates that free will wait for any ongoning communication)
but Free -> sendrecv is forbidden